test/regression/aarch64/instructions/sme.cc

```diff
@@ -1354,4 +1356,8 @@ INSTANTIATE_TEST_SUITE_P(AArch64, InstSme,
                          ::testing::ValuesIn(genCoreTypeSVLPairs(EMULATION)),
                          paramToString);
 
+#else
+GTEST_ALLOW_UNINSTANTIATED_PARAMETERIZED_TEST(InstSme);
```

Comment:
I assume this is something new we had to do in light of the GTEST version update?

Reply:
Yeah - if the SME test file is empty this clause is needed so that things don't break